src/ConvertToSarif/Properties/AssemblyInfo.cs

```diff
@@ -1,35 +1,11 @@
-﻿using System.Reflection;
-using System.Runtime.InteropServices;
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Here and below, clean up and DRY out AssemblyInfo files, and clean up their Title and Description texts as well.